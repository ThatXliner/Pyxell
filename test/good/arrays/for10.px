
a = ['a', 'q', 'x']
for x, y in a, 1..10 step -2, 3 do
    print x, y
