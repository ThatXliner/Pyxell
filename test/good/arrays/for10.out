x 1
a 4
